keyboards/coseyfannitutti/mysterium/rules.mk

```diff
@@ -0,0 +1,51 @@
+# MCU name
+MCU = atmega32a
+
+# Processor frequency
+F_USB = 16000000
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = USBasp
+
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+
+# Flash program via avrdude, but default command is not suitable.
+# You can use coseyfannitutti/discipline:default:program
+PROGRAM_CMD = avrdude -c usbasp -p m32 -U flash:w:$(BUILD_DIR)/$(TARGET).hex
```

Comment:
This isn't actually needed anymore. 

If you use `:usbasp` or `:flash`, it will actually ahndle this for you, automatically.

Reply:
Interesting... avrdude seems to know Caterina is 32U4, despite `atmega32u4` not appearing at all in avrdude.conf (its naming scheme is `m32u4`). Does `:usbasp` definitely work?

Reply:
:flash and :usbasp actually don't work for atmega32a for the same reason it didn't with toolbox previously. the avrdude command needs to be -p m32 or atmega32. This command passes atmega32a, which doesn't work.

Reply:
That's what I figured. Probably `-c avr109` completely ignores the MCU argument but most others don't.
Luckily this is pretty simple to solve; just add `AVRDUDE_MCU = m32` and `m328p` in mcu_selection.mk, and use that in the `:usbasp` target. I'll PR this in the morning :)

Reply:
When #7552 is merged, this can be removed 👍